.gitignore

```diff
@@ -173,3 +173,6 @@ grammars-Total/*.lark
 
 # Prod css doesn't have to be checked in, only leads to merge conflicts
 ./static/css/generated.css
+
+# hedy test cache
+.test-cache
```

Comment:
I see this is not checked in after all. Does that not mean my initial checkout + running the tests is going to take a long time?

Reply:
See my comment below https://github.com/hedyorg/hedy/pull/4574#issuecomment-1745072702 . The initial checkout + running the test will be slow-ish, but as you can run with `-n CPUs` and we are optimizing the parser caching too I think local testing will be fine. As always I'm happy to change my mind :)